orocos_kdl/src/chainiksolvervel_wdls.cpp

```diff
@@ -85,6 +85,11 @@ namespace KDL
         maxiter=maxiter_in;
     }
 
+    void ChainIkSolverVel_wdls::getSigma(Eigen::VectorXd& Sout)
+    {
+        Sout=S;
```

Comment:
Shouldn't we also add a `E_SIZE_MISMATCH` check her like in the other functions? Otherwise this might lead to implicit resizing which we try to avoid.
What is the main driver to not just return a `const Eigen::VectorXd&`?